src/runtime/components/navigation/CommandPalette.vue

```diff
@@ -223,7 +223,24 @@ export default defineComponent({
     const { results } = useFuse(query, commands, options)
 
     const groups = computed(() => ([
-      ...map(groupBy(results.value, command => command.item.group), (results, key) => {
+      ...(function groupBy (array, getKey) {
```

Comment:
Why not put the `groupBy` method inside the lodash utils?

Reply:
Based on daniel's advice, groupBy was only being used by `CommandPallete` Component, thats why, i can revert it back, if there's any plans to use it somewhere else

Reply:
I'm rewriting this slightly ... will push shortly

Reply:
i pushed something just now, but go for it 💪

Reply:
okay! sorry to overwrite https://github.com/nuxt/ui/pull/648/commits/7dd0c98c74f3d2aac3bb2f8b0d6938a782c74804